tox.ini

```diff
@@ -1,42 +1,8 @@
 [tox]
-envlist =
-    py{310,311,312}-test
-    py{310,311,312}-{poppydev,pysiafdev,astropydev,latest,stable}-test
-    py310-legacy-test
-    py{310,311,312}-{poppydev,pysiafdev}-cov
+envlist = docbuild
```

Comment:
Context for removing all the test environments:

Because the entire repository is replaced with stpsf, the tests that are running are stpsf tests.  This wasn't playing nicely with our CI as all tests labelled webbpsf no longer existed.  As this was intended to be the final release of WebbPSF, I decided to manually verify that the init change worked properly (as these automated tests will not need to run in the future because development is stopping in this repository).  I think that if we all take a look and are comfortable with testing the build manually, that suffices.  If this makes anyone uncomfortable I can spend time trying to get the CI functional, however I felt the benefits of that time investment were minimal.